raaghu-elements/src/rds-plan-switcher/rds-plan-switcher.tsx

```
@@ -0,0 +1,42 @@
+import React, { useEffect, useState }  from "react";
+import "./rds-plan-switcher.css";
+ 
+export interface RdsPlanSwitcherProps {
+  setIsPlanFree?: (isPlanFree: boolean) => void;
+}
+ 
+const RdsPlanSwitcher = (props:RdsPlanSwitcherProps) => {
+  const [isPlanFree, setIsPlanFree] = useState(true);
+
+  useEffect(() => {
+    props.setIsPlanFree && props.setIsPlanFree(isPlanFree);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)